src/lib/agent/method/dapps/degen/unruggable/method/createMemecoin.ts

```diff
@@ -0,0 +1,41 @@
+import { CreateMemecoinParams } from 'src/lib/agent/schema';
+import { stark, uint256 } from 'starknet';
+import { DECIMALS, Entrypoint } from '../constant';
+import { rpcProvider } from 'src/lib/agent/starknetAgent';
+import { decimalsScale, execute } from '..';
+
+export const createMemecoin = async (
+  params: CreateMemecoinParams,
+  privateKey: string
+) => {
+  try {
+    const salt = stark.randomAddress();
```

Comment:
this is bad because we need the owner to be determined by the user trough the prompt_ can you confirm what salt is about?

Reply:
mb this is for randomization